ontology/spdx-ontology.owl.xml

```diff
@@ -406,7 +417,9 @@ Known issues:</rdfs:comment>
                 </owl:unionOf>
             </owl:Class>
         </rdfs:range>
-        <rdfs:comment xml:lang="en">Licensing information that was discovered directly in the subject snippet. This is also considered a declared license for the snippet.</rdfs:comment>
+        <rdfs:comment xml:lang="en">Licensing information that was discovered directly in the subject snippet. This is also considered a declared license for the snippet.
+
+If the License Information in Snippet field is not present for a snippet, it implies an equivalent meaning to NOASSERTION.</rdfs:comment>
```

Comment:
Comments refer to the RDF terms, so use `licenseInfoInSnippet` instead of `License Information in Snippet`